### PR TITLE
Create standard text inputs

### PR DESCRIPTION
**Pull request by brunobc182**
# What

This __PR__ creates the `InputText` component and adds it to docs.

# Sample

image

### CODE REVIEW

**Comment by bezzocoaline** [comment]
:shipit:

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  "[javascript]": {
+    "editor.formatOnSave": true
+  },
+  "[javascriptreact]": {
+    "editor.formatOnSave": true
+  },
+  "editor.codeActionsOnSave": {
+    "source.fixAll.eslint": true
+  }
+}
```
Comment (AgtLucas) [inline .vscode/settings.json]: I'm not familiar with VSCode, but does this directory needs to be here, or can we ignore it?

File: packages/components/src/InputText/index.js [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+      <InputField
+        type="text"
+        id={inputId}
+        aria-labelledby={labelId}
+        placeholder={labelText}
+        {...rest}
+        isValidated={isValidated}
+        isInvalid={isInvalid}
+      />
+      <InputLabel id={labelId} htmlFor={inputId} isValidated={isValidated} isInvalid={isInvalid}>
```
Comment (AgtLucas) [inline packages/components/src/InputText/index.js]: What if we need to pass an attribute to the label?

File: packages/components/src/InputText/index.js [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+    ${props => inputValidationStyles({ ...props, defaultColor: props.theme.colors.moon500 })};
+  }
+
+  &:focus + ${InputLabel} {
+    ${props => labelValidationStyles({ ...props, defaultColor: props.theme.colors.uranus500 })};
+  }
+
+  &:focus,
+  &:not(:placeholder-shown) {
+    & + ${InputLabel} {
```
Comment (AgtLucas) [inline packages/components/src/InputText/index.js]: I think we can do just `label` instead, like so:

```css
& + label {}
```

File: packages/components/src/InputText/index.js [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+  right: 8px;
+`;
+
+const StyledIconCircleCheck = styled(IconCircleCheck)`
+  ${iconStyles}
+`;
+
+const StyledIconIconAlert = styled(IconAlert)`
+  ${iconStyles}
+`;
```
Comment (AgtLucas) [inline packages/components/src/InputText/index.js]: Suggestion:

I would get ride of this `css` and use it as an object instead, like so:

```js
const iconStyles = {
  position: 'absolute',
  top: '12px',
  right: '8px',
}
```

Then, I would spread it in the icon itself, like so:

```js
<IconAlert style={...iconStyles} />
```

What do you think?

File: .vscode/settings.json (hunk 1 above)
Comment (brunobc182) [reply .vscode/settings.json]: We can ignore it. :)

File: packages/components/src/InputText/index.js (hunk 2 above)
Comment (brunobc182) [reply packages/components/src/InputText/index.js]: We can create a `...labelAttrs` and let the user pass what he wants. Wdyt?

Comment (AgtLucas) [reply packages/components/src/InputText/index.js]: Sounds good to me! Maybe it should be named `labelProps` instead? I really don't know!

**Pull request merged**